lib/motsu/src/shims.rs

```diff
@@ -217,3 +217,115 @@ pub unsafe extern "C" fn account_codehash(_address: *const u8, dest: *mut u8) {
 
     std::ptr::copy(account_codehash.as_ptr(), dest, 32);
 }
+
+/// Returns the length of the last EVM call or deployment return result, or `0`
+/// if neither have happened during the program's execution. The semantics are
+/// equivalent to that of the EVM's [`RETURN_DATA_SIZE`] opcode.
+///
+/// [`RETURN_DATA_SIZE`]: https://www.evm.codes/#3d
+#[no_mangle]
+pub unsafe extern "C" fn return_data_size() -> usize {
+    // Default value, we do not use this function in our unit-tests.
```

Comment:
Note that this is not entirely accurate: we don't use them explicitly, but indeed the binary does include them, otherwise it wouldn't error when trying to compile.

I think it's fine to return zero here, but we should create a follow-up issue that tackles implementing the ones that make sense, like `return_data_size` and `read_return_data`.

Reply:
Created issue #156